src/pybind/rados/rados.pyx

```diff
@@ -224,6 +224,13 @@ class IsConnected(Error):
                 "RADOS is connected error (%s)" % message, errno)
 
 
+class BlocklistedError(Error):
```

Comment:
I'd rename to `ConnectionShutdown` to match the one in the RBD bindings (and the actual errno).

Reply:
Done.